app/controllers/api/v1/campaigns_controller.rb

```diff
@@ -31,7 +31,8 @@ def destroy
   end
 
   def find
-    @campaigns = Campaign.within_radius(250, params[:lat], params[:lng]).order_by_distance(params[:lat], params[:lng])
+    @campaigns = Campaign.within_radius(250, params[:lat], params[:lng])
+                         .order_by_distance(params[:lat], params[:lng])
```

Comment:
ooh order by distance. nice!

Reply:
Actually that was already on there. I just moved it down a line to fix the linter error :D

Reply:
oh haha